lib/Rx/SchedulerInterface.php

```diff
@@ -17,4 +17,11 @@ public function schedule($action);
      * @return DisposableInterface
      */
     public function scheduleRecursive($action);
+
+    /**
+     * Gets the current representation of now for the scheduler
+     *
+     * @return mixed
```

Comment:
Yeah, let's drop the functions exist check for it. I'm pretty certain it's safe.